pkg/stanza/README.md

```diff
@@ -0,0 +1,57 @@
+# pkg/stanza
+
+## Overview
+
+This module contains functionality for reading and parsing logs from traditional log media.
+Several of the collector's log receivers are based on this module.
+
+### History
+
+`stanza` was originally developed by observIQ as a standalone log agent. As a standalone agent, it had
+capabilities for reading, parsing, and exporting logs. It was donated to the OpenTelemetry project in 2021.
+Since then, it adapted to serve as the primary codebase for several log receivers in the OpenTelemetry Collector.
```

Comment:
```suggestion
Since then, it's adapted to serve as the primary codebase for several log receivers in the OpenTelemetry Collector.
```
Not sure if it should be `it's adapted` or `it's been adapted`

Reply:
Thanks, good catch